### DISCUSSION TITLE
Missing url type for Input component

### DISCUSSION THREAD

**Discussion by JairAviles** [twilio-labs/paste#3422] [General] [answered]
Currently, the `url` type prop is missing on the Input paste component [definition](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/input/src/Input.tsx#L273).

It would be nice to include it to save time on validation type. 

![url_type](https://github.com/twilio-labs/paste/assets/178258/7bf628c4-e061-41d0-a734-e768ba086778)

_**References**_:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url

**Comment by TheSisb** [comment]
Good catch, will add this in.

**Comment by nkrantz** [comment, marked as answer]
Keep an eye on [this PR](https://github.com/twilio-labs/paste/pull/3419) for the update to be released!
